dist/pym-loader.v1.js

```diff
@@ -7,15 +7,32 @@
 /** @module pym-loader */
 (function(requirejs, jQuery) {
     /**
+    * Create and dispatch a custom pym-loader event
+    *
+    * @method _raiseCustomEvent
+    * @inner
+    *
+    * @param {String} eventName
+    */
+   var _raiseCustomEvent = function(eventName) {
+     var event = document.createEvent('Event');
+     event.initEvent('pym-loader:' + eventName, true, true);
+     document.dispatchEvent(event);
+   };
+
+    /**
     * Initialize pym instances if Pym.js itself is available
     *
     * @method initializePym
     * @instance
     *
     * @param {String} pym Pym.js loaded library.
     */
-    var initializePym = function(pym) {
+    var initializePym = function(pym, ignoreEvent) {
```

Comment:
Name would be more clear if something like doNotRaiseEvents
ignoreEvents implies that it is ignoring events from another place